apps/io-services-cms-webapp/src/utils/apim.ts

```diff
@@ -0,0 +1,46 @@
+import { ApiManagementClient } from "@azure/arm-apimanagement";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { pipe } from "fp-ts/lib/function";
+import * as TE from "fp-ts/lib/TaskEither";
+import {
+  ApimRestError,
+  getSubscription,
+  getUser,
+  getUserGroups,
+  parseOwnerIdFullPath,
+} from "../apim_client";
+import { Delegate } from "./service_review_proxy";
+
+export const getDelegateFromServiceId = (
+  apimClient: ApiManagementClient,
+  apimResourceGroup: string,
+  apim: string,
+  serviceId: NonEmptyString
+): TE.TaskEither<ApimRestError, Delegate> =>
+  pipe(
+    getSubscription(apimClient, apimResourceGroup, apim, serviceId),
+    TE.chain((subscription) =>
+      pipe(
+        parseOwnerIdFullPath(subscription.ownerId as NonEmptyString),
+        (ownerId) => getUser(apimClient, apimResourceGroup, apim, ownerId),
+        TE.chainW((user) =>
+          pipe(
+            getUserGroups(
+              apimClient,
+              apimResourceGroup,
+              apim,
+              user.name as NonEmptyString
+            ),
+            TE.map(
+              (userGroups) =>
+                ({
+                  name: `${user.firstName} ${user.lastName}`,
+                  email: user.email,
+                  permissions: userGroups.map((group) => group.name),
+                } as Delegate)
```

Comment:
Can we avoid manual casting? This could lead to an issue if we update the `Delegate` definition in the future.
Consider these alternatives:
* make the object to be in the shape of a `Delegate` (so that no cast is required)
* decode the object (to enforce strict validation of data)
* relax the `Delegate` definition (if you think it's unnecessarily strict and unhandy to use)